tools/mull-cxx/CLIOptions.cpp

```diff
@@ -257,6 +258,10 @@ std::vector<std::unique_ptr<Reporter>> ReportersCLIOptions::reporters(ReporterPa
       reporters.emplace_back(new mull::SQLiteReporter(diagnostics, directory, name));
     } break;
     case ReporterKind::Elements: {
+      if (!params.compilationDatabaseAvailable) {
```

Comment:
I'm not 100% sure if this should go here, but we can find a better place eventually.

Reply:
Me too 😄